src/python/example_parallel.py

```diff
@@ -5,7 +5,7 @@
 from timeit import default_timer as timer
 import os
 
-robot = rd.Robot("res/models/arm.urdf", "arm", False)
+robot = rd.Robot("res/robots/arm.urdf", "arm", False)
```

Comment:
> I guess these are left-overs from another branch right?

Same here.

Reply:
Yes. Sorry, I had a few things staged and I wrote 'commit' a bit too fast. This is from urdf_library...

Reply:
Can you revert them? (or just do a new commit that is undoing them)